web/lib/src/dom/console.dart

```diff
@@ -47,35 +47,86 @@ external $Console get console;
 @JS('console')
 extension type $Console._(JSObject _) implements JSObject {
   @JS('assert')
-  external void assert_(
-    JSAny? data, [
+  external void assert_([
```

Comment:
Breaking change. :(

Reply:
Keep it or make it not optional if argument is first?

Reply:
Let's keep it but update the changelog and pubspec for web `2.0.0` for now. Depending on what we may decide for versioning, we can modify as needed later.